src/PluginForm/Bluesnap/PaymentMethodAddForm.php

```diff
@@ -0,0 +1,176 @@
+<?php
+
+namespace Drupal\commerce_bluesnap\PluginForm\Bluesnap;
+
+use Drupal\commerce\InlineFormManager;
+use Drupal\commerce_bluesnap\ApiService;
+use Drupal\commerce_payment\PluginForm\PaymentMethodAddForm as BasePaymentMethodAddForm;
+
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Routing\RouteMatchInterface;
+
+use Psr\Log\LoggerInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * The BlueSnap PaymentMethodAddForm.
+ *
+ * @package Drupal\commerce_bluesnap\PluginForm\Bluesnap
+ */
+class PaymentMethodAddForm extends BasePaymentMethodAddForm {
```

Comment:
**CHANGE REQUEST**

This is the form specifically for the Hosted Payment Fields gateway, eventually we will have to end up with multiple as they each have different form fields: 

* `HostedPaymentFieldsPaymentMethodAddForm`
* `AchPaymentMethodAddForm`
* `EmbeddedCheckoutPaymentMethodAddForm`

etc.